Compile fails with g++ on Ubuntu rolling

Compilation fails in a docker image using ubuntu:rolling. With the message

```
63.29 /usr/local/nextsimdg/physics/src/modules/SMUIceAlbedo.cpp:18:100: error: return type 'class std::tuple<double, double>' is incomplete
63.29    18 | std::tuple<double, double> SMUIceAlbedo::albedo(double temperature, double snowThickness, double i0)
```

The solution appears to be to add ``#include <tuple>`` to the ``IIceAlbedo.hpp`` header file.

Minimal Dockerfile is

```
FROM ubuntu:rolling

RUN apt-get update --fix-missing
RUN apt-get install -y --no-install-recommends \
ca-certificates \
git \
cmake \
netcdf-bin \
libnetcdf-c++4-dev \
libboost-all-dev \
libeigen3-dev \
g++ \
make

WORKDIR /usr/local

RUN git clone -b develop https://github.com/nextsimdg/nextsimdg.git

WORKDIR /usr/local/nextsimdg

RUN git pull \
        && rm -rf build \
        && mkdir -p build \
        && cd build \
        && CMAKE_BUILD_TYPE=Debug cmake .. \
        && make -j 8\
        && cp nextsim /usr/local/bin/
```

--- a/physics/src/modules/include/IIceAlbedo.hpp
+++ b/physics/src/modules/include/IIceAlbedo.hpp
@@ -1,7 +1,7 @@
 /*
  * @file IIceAlbedo.hpp
  *
- * @date Aug 10, 2021
+ * @date Aug 10, 2023
  * @author Tim Spain <timothy.spain@nersc.no>
  * @author Einar Örn Ólason <einar.olason@nersc.no>
  */

--- a/physics/src/modules/include/IIceAlbedo.hpp
+++ b/physics/src/modules/include/IIceAlbedo.hpp
@@ -9,6 +9,7 @@
 #define IICEALBEDO_HPP
 
 #include "include/Time.hpp"
+#include <tuple>
 
 namespace Nextsim {
 //! The interface class for ice albedo calculation.

--- a/physics/src/modules/include/IIceAlbedo.hpp
+++ b/physics/src/modules/include/IIceAlbedo.hpp
@@ -1,8 +1,9 @@
 /*
  * @file IIceAlbedo.hpp
  *
- * @date Sep 28, 2021
+ * @date Aug 10, 2021
  * @author Tim Spain <timothy.spain@nersc.no>
+ * @author Einar Örn Ólason <einar.olason@nersc.no>
  */
 
 #ifndef IICEALBEDO_HPP

